File: golang/002.go
package main

func findMaxMin(arr []int) []int {
	max, min := arr[0], arr[0]
	for _, ch := range arr {
		if ch > max {
			max = ch
		}
		if ch < min {
			min = ch
		}
	}
	return []int{max, min}
}

// func main() {
// 	fmt.Println(findMaxMin([]int{1, 4, 7, 2, 3, 4, 3, 6, 8}))
// }

File: python/002.py
def getMaxMin(arr):
    max, min = arr[0],arr[0]
    for x in arr[1:]:
        if x > max:
            max = x
        if x < min:
            min = x
    return [max, min]

print(getMaxMin([3,4,6,7,3,2,4,6]))